### PR TITLE
expose dialog open state from service

### PR DESCRIPTION
**Pull request by shaharkazaz**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: projects/ngneat/dialog/src/lib/dialog.service.ts [hunk 1]
```diff
@@ -26,7 +26,19 @@ export class DialogService {
   private document = inject(DIALOG_DOCUMENT_REF);
   private globalConfig = inject(GLOBAL_DIALOG_CONFIG);
 
-  dialogs: DialogRef[] = [];
```
Comment (NetanelBasal) [inline projects/ngneat/dialog/src/lib/dialog.service.ts:29]: This is a breaking change

Comment (shaharkazaz) [reply projects/ngneat/dialog/src/lib/dialog.service.ts:29]: Correct. I can deprecate it and make an adapter to avoid the breaking change

Comment (NetanelBasal) [reply projects/ngneat/dialog/src/lib/dialog.service.ts:29]: or you can just leave it as an array

Comment (shaharkazaz) [reply projects/ngneat/dialog/src/lib/dialog.service.ts:29]: I think it makes more sense as a map no?

**Pull request merged**